### PR TITLE
Add CHD image support for PSX

### PR DESCRIPTION
**Pull request by zakk4223**

### CODE REVIEW

**Comment by bootsector** [comment]
OMG! 😱

**Comment by thehughhefner** [comment]
Nice job Zakk!

**Comment by makigumo** [comment]
Well done! Could you please use tabs for indentation like in the rest of the file?

File: support/psx/psx.cpp [hunk 1]
```diff
@@ -107,6 +123,35 @@ static void unload_cue(toc_t *table)
...
+{
+
+  unload_chd(table);
+  chd_error err = mister_load_chd(filename, table);
+  if (err != CHDERR_NONE)
+  {
+    return 0;
+  }
+
+  for(int i = 0; i < table->last; i++)
```
Comment (tjanas) [inline support/psx/psx.cpp:138]: Does the compiler optimize out the i++ to make it ++i in machine code?

Comment (sorgelig) [reply support/psx/psx.cpp:138]: table->last is not the last after processing, it's total number of tracks. If you are talking about it.

**Pull request merged**